webserver/external/messybrainz.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import ujson
+
+def get_messybrainz_id(querydata):
+    """
+    This function queries messybrainz to get a msid according to the 'artist/title' present in the item metadata
+    Args:
+        querydata: A dictionary containing at least artis and song title
+
+    Returns:
+        msid: string messybrainz id -> if it is found in the messybrainz database
+        None: if no reference is fount in messybrainz database
+
+    """
+    msurl = 'https://messybrainz.org/submit'
+    if isinstance(querydata, list):
+        if 'artist' in querydata[0].keys() and 'title' in querydata[0].keys():
+            jsondata = ujson.dumps(querydata)
+            r = requests.post(msurl, data=jsondata)
+            if r.status_code == 200:
```

Comment:
requests has a neat method, `r.raise_for_status()` (http://docs.python-requests.org/en/master/api/#requests.Response.raise_for_status) which will do nothing if the status code is 200, otherwise raise an exception.